modules/log6x-oc-explain.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: CONCEPT
+[id="log6x-oc-explain_{context}"]
+
+= Using the `oc explain` command
+
+The `oc explain` command is an essential tool in the OpenShift CLI `oc` that provides detailed descriptions of the fields within Custom Resources (CRs). This command is invaluable for administrators and developers who are configuring or troubleshooting resources in an OpenShift cluster.
+
+== Resource Descriptions
+`oc explain` offers in-depth explanations of all fields associated with a specific object. This includes standard resources like pods and services, as well as more complex entities like statefulsets and custom resources defined by Operators.
+
+For example, to view the documentation for the `outputs` and `options` fields of the `ClusterLogForwarder` custom resource, you can use:
+
+[source,terminal]
+----
+$ oc explain clusterlogforwarders.logging.openshift.io.spec.outputs
+$ oc explain clusterlogforwarders.logging.openshift.io.spec.outputs.options
```

Comment:
These two APIs are wrong. They were in Logging 5, but not more valid in Logging 6 as they are: clusterLogForwarder.observability.openshift.io instead of clusterlogforwarders.logging.openshift.io